<!DOCTYPE html>
<html lang="en">

<head>
	<meta charset="utf-8">
	<title>MoleOnline | MOLE 2.0 and MOLEonline 2.0 Algorithm Explanation</title>
	<meta name="description"
		content="Method section describes the channel detection algorithm used in MOLE 2.0 and MOLEonline 2.0 using Voronoi diagram representation for the protein followed by spliting to smaller cavity diagrams and finding path between starting and ending points with Dijk">
	<meta name="author" content="Ing. Ondřej Hanák; http://ondrejhanak.cz">
	<link rel="shortcut icon" href="\images/favicon.ico" type="image/icon">
	<!-- <script type="text/javascript" src="\js/old-web/jsloader.js"></script> -->
	<link rel="stylesheet" type="text/css" media="screen" href="../css/init/cssloader.css">
	<link rel="stylesheet" href="../css/init/print.front.css" type="text/css" media="print">
	<!-- <link rel="stylesheet" href="../css/init/LiteMol-plugin.css" type="text/css"> -->
	<link rel="stylesheet" href="../css/init-styles.css" type="text/css">
	<!-- <script src="\js/provided.js?version=<!UI-VERSION!>"></script>
	<script src="\js/scripts.js?version=<!UI-VERSION!>"></script>   -->

	<!-- <script src="../js/scripts.js?version=R1.1.9.1"></script> -->
	<script src="../js/init-scripts.js?version=R1.1.9.1"></script>
	<!-- Global site tag (gtag.js) - Google Analytics -->
	<script async src="https://www.googletagmanager.com/gtag/js?id=UA-4336835-7"></script>
	<script>
		window.dataLayer = window.dataLayer || [];
		function gtag() { dataLayer.push(arguments); }
		gtag('js', new Date());

		gtag('config', 'UA-4336835-7');
	</script>
	<style>
		.dropdown:hover .dropdown-menu {
			display: block;
		}

		.dropdown-menu {
			margin-top: 0;
		}
	</style>
</head>

<body>

	<div id="page">

		<div class="card shadow-sm mb-4 sticky-top">
			<div class="card-body d-flex align-items-center" style="position: relative;">
				<div class="header-gradient-background"></div>
				<div class="header-gradient-text fs-3 fw-bold px-3" style="margin-right: 28rem;">
					<img src="\images/favicon.ico" alt="M" class="header-icon">
					oleOnline
				</div>
				<ul class="nav nav-pills nav-fill flex-grow-1">
					<li class="nav-item">
						<a class="nav-link" href="/">Home</a>
					</li>
					<li class="nav-item dropdown">
						<a class="nav-link active dropdown-toggle" aria-current="page" href="documentation.html"
							id="navbarDropdown" aria-expanded="false">Documentation</a>
						<ul class="dropdown-menu w-100" aria-labelledby="navbarDropdown">
							<li><a class="dropdown-item active" href="#">Method</a></li>
							<li><a class="dropdown-item" href="tour.html">Tour</a></li>
							<li><a class="dropdown-item" href="faq.html">FAQ</a></li>
							<li><a class="dropdown-item" href="updates.html">Updates</a></li>
						</ul>
					</li>
					<li class="nav-item">
						<a class="nav-link" href="about.html">About</a>
					</li>
					<li class="nav-item">
						<a class="nav-link" href="news.html">News</a>
					</li>
				</ul>
			</div>
		</div>

		<div class="card p-4" style="border: var(--bs-card-border-width) solid rgba(205, 205, 205, 0.18);">
			<h3>Methods and other Details</h3>
			<h4>MOLE<em>online</em> algorithm </h4>
			<p>
				The computation of the tunnel is performed in several steps (see Figure 1)
				described in more detail in <a target="_blank" href="https://doi.org/10.1186/1758-2946-5-39">MOLE 2.0
					publication</a>:
			</p>
			<ol>
				<li>
					computation of the Delaunay triangulation/Voronoi diagram of the atomic centers,
				</li>
				<li>
					construction of the molecular surface,
				</li>
				identification of cavities,
				</li>
				<li>
					identification of possible channel start points,
				</li>
				<li>
					identification of possible channel end points,
				</li>
				<li>
					localization of channels using ,
				</li>
				<li>
					filtering of the localized channels.
				</li>
			</ol>
			<figure class="figure">
				<img class="img-fluid figure-img" src="\images/method_MOLE_scheme.png"
					alt="Scheme of the channel computation by MOLE">
				<figcaption>Figure 1. Scheme of the channel computation by MOLE.</figcaption>
			</figure>
			<h4>Calculation of channels</h4>
			<p>
				The channel computation is finished after the definition of starting and ending points for all of
				cavities,
				the Dijkstra’s Shortest Path Algorithm is used to find the channels between all pairs of starting and
				ending
				points (or the molecular surface if the algorithm reaches it before it finds the channel exit). The edge
				weight
				function used in the algorithm takes into account the distance to the surface of the closest vdW sphere
				and the edge
				length.
			</p>
			<p>
				The channel centerline is represented as a 3D natural spline representation defined by the Voronoi
				diagram
				vertices that form the path found by the Dijkstra‘s algorithm.
			</p>
			<p>
				The algorithm might find duplicate channels. Therefore, in the last post-processing step,
				the duplicate channels are removed according to Channel parameters.
			</p>
			<h4>Pore calculation</h4>
			<p>
				Pores are calculated differently than channels in the classical MOLE computation and is divided into two
				steps:
			</p>
			<ul style="list-style-type:lower-roman">
				<li>finding of membrane region and</li>
				<li>calculation of the pore.</li>
			</ul>
			<p>
				The pore computation starts with defining of the membrane region of the molecule. Firstly,
				MOLE is trying to find the structure on Orientations of Proteins in Membranes - OPM
				database (<a target="_blank" href="http://opm.phar.umich.edu/">http://opm.phar.umich.edu/</a>;
				Lomize,M.A., Lomize,A.L., Pogozheva,I.D. and Mosberg,H.I. (2006)
				OPM: Orientations of Proteins in Membranes Database. <i>Bioinformatics</i>, <b>22</b>, 623–625).
			</p>
			<p>
				If the structure cannot be found in the OPM database the MOLE continue in searching the membrane region
				by using the MEMEMBED program (Nugent,T. and Jones,D.T. (2013) Membrane protein orientation and
				refinement
				using a knowledge-based statistical potential. <i>BMC Bioinformatics</i>, <b>14</b>, 276) which is able
				to predict
				the probable membrane region of the protein molecule.
			</p>
			<p>
				When the membrane region definition is finished the pore end points are computed. The option <b>Beta
					structure</b>
				serves as the parameter for refinement of the membrane region prediction. The prediction of the membrane
				region
				highly depends on the secondary structure of the studied molecule.
			</p>

			<h3>Further reading</h3>
			<ul style="list-style-type: disc !important;">
				<li>
					Pravda L., Berka K., Sehnal D., Otyepka M., Svobodová Vařeková R., Koča J.Koča J, Svobodová Vařeková
					R,
					Pravda L, Berka K, Geidl S, Sehnal D, Otyepka M.
					<a href="https://www.springer.com/cn/book/9783319473871" target="_blank">Detection of Channels.
						Structural
						Bioinformatics Tools for Drug Design.</a> 2016; Cham: Springer. 59–69.
				</li>
				<li>
					Simões T., Lopes D., Dias S., Fernandes F., Pereira J., Jorge J., Bajaj C., Gomes A.
					<a href="http://dx.doi.org/10.1111/cgf.13158" target="_blank">Geometric detection algorithms for
						cavities
						on protein surfaces in molecular graphics: a survey.</a> Comput. Graph. Forum. 2017.
					<i>Excellent review not
						only about different methods for channel analysis but also about the theory behind</i>.
				</li>
				<li>
					Edelsbrunner, H., &amp; Mücke, E. P. <a href="https://doi.org/10.1145/174462.156635"
						target="_blank">
						Three-dimensional alpha shapes</a>. ACM Transactions on Graphics,
					13(1), 43–72. 1994. <i>Mathematical background of Voronoi diagrams construction used in MOLE
						algorithm</i>.
				</li>
			</ul>

			<h3>Other Details</h3>

			<h4>Origins</h4>
			<p>
				This option allows choosing of the start point by clicking to the ball which is pre-calculated inside
				the structure after upload of PDB code or your own structure from the file. After the click to the ball
				the selection will be created and this selection can be used as the start point. Similarly, the cavity
				may be also defined as the starting point but is necessary to first choose the cavity residues by
				clicking to the structure with CTRL+click and choose the specific amino acids localized in the cavity
				(Figure 2).
			</p>

			<figure class="figure">
				<img class="img-fluid figure-img" src="\images/method_other_details_origins.png"
					alt="The visualization of the Origins">
				<figcaption>Figure 2. The visualization of the Origins – showed as a violet balls in the structure of
					the cytochrome P450 3A4 (PDB ID: 1TQN).</figcaption>
			</figure>

			<p>
				Another option is just selection of several residues by clicking to the structure and put this selection
				to the starting or ending point box (or other fields which can work with the selection of residues).
				In the same way, the sequence viewer can be also used for the selection of the starting point.
			</p>

			<p>
				During the clicking to the individual residues the center of mass is calculated gradually and it is
				displayed
				as a clickable ball.
			</p>

			<h3>Visualization of results</h3>

			<h4>Radius of channel/pore</h4>
			<p>The MOLEonline offers three representations of the channel/pore radius (Figure 3):</p>
			<ol style="list-style-type:lower-alpha">
				<li><b>Radius</b> – Radius of sphere within channel limited by three closest atoms</li>
				<li><b>Free Radius</b> – Radius of sphere within channel limited by three closest main chain atoms in
					order to allow sidechain flexibility</li>
				<li><b>BRadius</b> – Radius + RMSF calculated from B-factors of residues within individual layers <img
						class="img-fluid" src="\images/method_other_rmsf.png" /></li>
			</ol>

			<figure class="figure">
				<img class="img-fluid figure-img" src="\images/method_other_detail_of_channel_profile.png"
					alt="Detail of the channel profile">
				<figcaption>Figure 3. Detail of the channel profile. Profile can be visualized for various properties.
				</figcaption>
			</figure>

			<h4>Calculation of physico-chemical properties</h4>
			<p>
				When the tunnel is computed the set of amino acids (side chains) lining the channel can be used for the
				physico-chemical properties computation (Figure 4). According to side chains composition, the set of
				physico-chemical properties is calculated (for lining amino acids side chains only):
			</p>
			<p>
				<b>Charge</b> is calculated as a sum of charged amino acid residues (ARG, LYS, HIS = +1; ASP, GLU = -1)
			</p>
			<p>
				<b>Hydropathy</b> is calculated as an average of hydropathy index assigned to residues according
				to the method of Kyte and Doolitle <a target="_blank"
					href="https://www.ncbi.nlm.nih.gov/pubmed/7108955">(Kyte, J. and Doolittle, R.F. (1982) A Simple
					Method for
					Displaying the Hydropathic Character of a Protein. <i>J. Mol. Biol</i>., <b>157</b>, 105–132</a>)
			</p>
			<p>
				Hydropathy index is connected to hydrophilicity/hydrophobicity of amino acids (most hydrophilic is ARG =
				-4.5; most hydrophobic ILE = 4.5).
			</p>

			<p>
				<b>Hydrophobicity</b> is calculated as an average of normalized hydrophobicity scales by Cid et al.
				(<a target="_blank" href="https://www.ncbi.nlm.nih.gov/pubmed/1518784">Cid, H., Bunster, M., Canales, M.
					and Gazitúa, F. (1992) Hydrophobicity and structural classes in proteins.
					<i>Protein Eng. Des. Sel</i>., <b>5</b>, 373–375</a>).
			</p>

			<p>
				Most hydrophilic amino acid according to hydrophobicity value is GLU (-1.140) and the most hydrophobic
				is ILE (1.810).
			</p>

			<p>
				<b>Polarity</b> is calculated as an average of amino acid polarities assigned according to the method
				of Zimmerman et al. (<a target="_blank" href="https://www.ncbi.nlm.nih.gov/pubmed/5700434">Zimmerman,
					J.M., Eliezer, N. and Simha, R. (1968) The Characterization of Amino Acid Sequences
					in Proteins by Statistical Methods. <i>J. Theor. Biol</i>., <b>21</b>, 170–201</a>).
			</p>

			<p>
				Polarity ranges from completely nonpolar amino acids (ALA, GLY = 0.00) through polar residues (e.g. SER
				= 1.67) towards charged residues (GLU = 49.90, ARG = 52.00).
			</p>

			<p>
				<b>Lipophilicity</b> <i>(logP-scale)</i> is calculated as octanol/water partition coefficients of Cβ
				fragments of side-chains and mainchain (-0.86) via <a target="_blank"
					href="www.chemicalize.org">www.chemicalize.org</a>.
			</p>
			<p>
				LogP values ranges from lipophobic fragments (ASN -1.03; SER -0.52; GLN -0.33; THR -0.16) through acids
				(ASP -0.22; GLU 0.48) and bases (ARG -0.08; HIS -0.01; LYS 0.7), to sulphur-containing (CYS 0.84; MET
				1.48), aliphatic (ALA 1.08; VAL 1.8; PRO 1.8; LEU 2.08; ILE 2.24) to most lipophilic aromatic residues
				(TYR 2.18; PHE 2.49; TRP 2.59). GLY has no sidechain fragment and therefore its value is set to 0.
			</p>

			<p>
				<b>Lipophilicity</b> <i>(logD-scale)</i> is calculated as octanol/water distribution coefficients of Cβ
				fragments of side-chains and mainchain (-0.86) at pH 7.4 via <a target="_blank"
					href="www.chemicalize.org">www.chemicalize.org</a>. Distribution coefficient take into account
				ionization of compounds.
			</p>
			<p>
				LogD values ranges from highly lipophobic charged fragments (ASP -3.00; ARG -2.49; GLU -2.12; LYS -1.91)
				through polar ones (ASN -1.03; SER -0.52; GLN -0.33; THR -0.16, HIS -0.11) to sulphur-containing (CYS
				0.84; MET 1.48), aliphatic (ALA 1.08; VAL 1.8; PRO 1.8; LEU 2.08; ILE 2.24) to most lipophilic aromatic
				residues (TYR 2.18; PHE 2.49; TRP 2.59). GLY has no sidechain fragment and therefore its value is set to
				0.
			</p>

			<p>
				<b>Solubility</b> <i>(logS-scale)</i> is calculated as water solubility of Cβ fragments of side-chains
				and mainchain (0.81) at pH 7.4 via <a target="_blank"
					href="www.chemicalize.org">www.chemicalize.org</a>. Our estimated logS value is a unit stripped
				logarithm (base 10) of the solubility measured in mol/liter and it is a measure how well can be
				individual residues interacting with water molecules.
			</p>
			<p>
				LogS values ranges from lowest solubility of aromatic residues (TRP -2.48: PHE -1.81; TYR -1.44;)
				followed by aliphatic (LEU -1.79; ILE -1.85; PRO -1.3; VAL -1.3; ALA 0.59) and sulphur-containing
				residues (MET -0.72; CYS 0.16) up to polar ones (HIS -0.2; GLN 0.13; ASN 0.54; THR 0.77; SER 1.11)
				whereas the most water molecules will attract charged residues (LYS 1.46; ARG 1.63; GLU 2.23; ASP 2.63).
				GLY has no sidechain fragment and therefore its value is set to 0.
			</p>

			<p>
				<b>Mutability</b> is calculated as an average of relative mutability index (<a target="_blank"
					href="https://www.ncbi.nlm.nih.gov/pubmed/1633570">Jones,D.T., Taylor,W.R.
					and Thornton,J.M. (1992) The Rapid Generation of Mutation Data Matrices from Protein Sequences.
					<i>Bioinformatics</i>,
					<b>8</b>, 275–282</a>). Relative mutability is based on empirical substitution matrices between
				similar protein sequences.
			</p>

			<p>
				It is high wherever amino acid can be easily substituted for another e.g. in case of small polar amino
				acids (SER = 117, THR = 107, ASN = 104) or in case of small aliphatic amino acids (ALA = 100, VAL = 98,
				ILE = 103). On the other hand it is low when amino acid plays important structural role such as in case
				of aromatic amino acids (TRP = 25, PHE = 51, TYR = 50) or in case of special amino acids (CYS = 44, PRO
				= 58, GLY = 50). Specific example of amino acid with low relative mutability is the most abundant amino
				acid /LEU = 54) which has the highest probability to mutate back to itself.
			</p>

			<p>
				<b>Ionizable</b> residues can be also viewed in the channel profile or directly as selection on the
				visualized structure.
			</p>

			<figure class="figure">
				<img class="img-fluid figure-img" src="\images/method_other_computed_hphys_chem_props.png"
					alt="Example of computed physicochemical properties">
				<figcaption>Figure 4. Example of computed physicochemical properties (Pore of structure 1K4C).
				</figcaption>
			</figure>

			<h3>Annotations</h3>
			<p>
				You can contribute to the ChannelsDB database (<a target="_blank"
					href="http://ncbr.muni.cz/ChannelsDB/">http://ncbr.muni.cz/ChannelsDB/</a>) by Annotation button in
				the upper left corner of the visualization window. Annotation can be easily written to the boxes in the
				Annotation window and channels selected from those calculated within the window (Figure 5).
			</p>

			<figure class="figure">
				<img class="img-fluid figure-img" src="\images/method_other_annotate_form.png"
					alt="Annotation window for ChannelsDB database">
				<figcaption>Figure 5. MOLEonline Annotation window for contribution to ChannelsDB database.</figcaption>
			</figure>

		</div>
	</div>

	<div id="ajaxspinner"></div>

	<script src="../MOLEOnline_Init-Core.js?version=R1.1.9.1"></script>
</body>

</html>